src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -2421,8 +2548,8 @@ subroutine gsi_fv3ncdf_write_v1(filename,varname,var,mype_io,add_saved)
        if(add_saved)then
           allocate( workb2(nlon_regional,nlat_regional))
           allocate( worka2(nlat,nlon))
-!  for being now only lev between (including )  2 and nsig+1 of work_b (:,:,lev) 
-!  are updated
+!clt  for being now only lev between (including )  2 and nsig+1 of work_b (:,:,lev) 
```

Comment:
it is better to use the lines in the master (namely, remove those clt string in the beginnings of the comment lines).

Reply:
Removed the "ctl"